package.json

```diff
@@ -40,6 +40,7 @@
         "eslint-plugin-import": "^2.8.0",
         "mocha": "^5.0.0",
         "nodemon": "^1.14.11",
+        "preact-jsx-chai": "^2.2.1",
```

Comment:
almost that's that right package and it's in devDependencies however this needs to be in the client>package.json this is the root>package.json